public/images/waveform--white.svg

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
```

Comment:
a green version of this waveform already exists, shouldn't we show the same one?

Reply:
![image](https://user-images.githubusercontent.com/90180417/184313264-bdf3b871-7649-475a-bf0c-ce936de249ad.png)

Reworking the existing waveform to use currentColor produces too many potential side-effects